umbraco/Umbraco-CMS
# V14: Additional blueprint endpoints

## Details
- Fixed `/item` endpoint to return 0 items when no items' keys are passed in, as opposed to returning all items (_there is a task to align this behaviour across all item endpoints_);
- Added 2 new blueprint endpoints:
  - GET `/umbraco/management/api/v1/tree/document-blueprint/ancestors` - used for breadcrumbs.
  - GET `/umbraco/management/api/v1/document-type/{id}/blueprint` - used when creating new content, to show the blueprints available for a document type.
- Fixed some test cases + added some more; 

## Test
- Create some blueprints;
- Test the 3 endpoints:
  - Make sure that you can get the items you expect through the `/item` endpoint and that you get 0 items when no keys are passed;
  - Pass the blueprint key to the `/ancestors` endpoint and verify that you get the folder and the item in response;
  - After you have created several blueprints for a `/document-type`, see if you get the correct items through the document type endpoint.
